sections/03_categorize.md

```diff
@@ -19,7 +19,7 @@ tackle any of them? Are there example approaches whereby deep learning is
 already having a transformational impact? I (Casey) have added some sections
 below where I think we could contribute to the field with our discussion.*
 
-### Major challenges
+### Major Areas of Existing Contributions
```

Comment:
We should settle on one. I don't have a preference either way. What do you prefer? I suggest we make one quick PR to go through and match case as soon as we decide.